client/src/components/CreatePost/Form/Form.js

```diff
@@ -109,26 +143,36 @@ const ModalComponent = ({ setCurrentStep, onClose }) => {
         <TabPane tab="Offering Help" key="1">
           <OfferHelp
             formData={formData}
-            handleSubmit={handleSubmit}
+            handleSubmit={(e) => handleSubmit(e)}
             handleFormData={handleFormData}
             renderError={renderError}
             addTag={addTag}
             selectedTags={formData.tags}
+            onExpirationChange={setExpiration}
+            onShareWithChange={setShareWith}
           />
         </TabPane>
         <TabPane tab="Requesting Help" key="2">
           <AskHelp
             formData={formData}
-            handleSubmit={handleSubmit}
+            handleSubmit={(e) => handleSubmit(e)}
```

Comment:
any reason why you add the `(e) =>` arrow function? I prefer to just use the handler directly, with hooks it doesn't make a difference but with class components you avoid recreating the handler functions for every render

Reply:
just my preference but i don't really mind changing it back since it'll match the rest of the code. I changed it back!